src/components/ListPage/ListPage.ts

```diff
@@ -24,6 +25,7 @@ export namespace ListPage {
     getFiltersFromURL: (filterTypes: FilterType[]) => ActiveFilter[];
     setFiltersToURL: (filterTypes: FilterType[], filters: ActiveFilter[]) => ActiveFilter[];
     filtersMatchURL: (filterTypes: FilterType[], filters: ActiveFilter[]) => boolean;
+    sortFields: () => SortField<any>[];
```

Comment:
But here I have a `code smell` that makes me think that I am approaching things correctly.

Reply:
I do need this one in order to remove duplicate LOCs in [(Service|Workload|App|Istio)ListPage](https://github.com/kiali/kiali-ui/pull/714/files#diff-0e102c782926f54acf5b4ec848eee25cR14)